lib/stat_tracker.rb

```diff
@@ -59,4 +59,77 @@ def read_game_teams_csv(location_path)
       end
       game_team_data
    end
+
+   def count_of_teams
+      @data_teams.count  
+   end
+
+   def best_offense
+      convert_team_id_to_name(highest_average_team_id(team_stats))
+   end
+
+   def worst_offense
+      convert_team_id_to_name(lowest_average_team_id(team_stats))
+   end
+
+   def highest_scoring_visitor
+      convert_team_id_to_name(highest_average_team_id(team_stats_hoa("away")))
```

Comment:
I would adjust the formating of this method. I don't know if it's just GitHub or if there's a line between the name of the method and the beginning of execution. Either way, easy fix!